sql/report_util_functions.sql

```diff
@@ -796,6 +796,31 @@ BEGIN
 END$$
 DELIMITER ;
 
+-- patientIsPregnant
+
+DROP FUNCTION IF EXISTS patientIsPregnant;
+
+DELIMITER $$
+CREATE FUNCTION patientIsPregnant(
+    p_patientId INT(11)) RETURNS TINYINT(1)
+    DETERMINISTIC
+BEGIN 
+    DECLARE patientPregnant TINYINT(1) DEFAULT 0;
+
+    DECLARE uuidpatientIsPregnant VARCHAR(38) DEFAULT "279583bf-70d4-40b5-82e9-6cb29fbe00b4";
```

Comment:
@IipperMangesh please use uppercase 'P' on the variable's name (i.e. `uuidPatientIsPregnant`)?